### PR TITLE
Remove e2e reading

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- remove yaml support for end-to-end stories
- remove markdown support for end-to-end stories
- remove the creation of additional NLU training data from domain / stories

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: rasa/core/schemas/stories.yml [hunk 1]
```diff
@@ -45,16 +45,6 @@ mapping:
               action:
                 type: "str"
                 allowempty: False
-          - type: "map"
-            mapping:
-              user:
```
Comment (wochinge) [inline rasa/core/schemas/stories.yml:50]: we might need to keep this for end-to-end tests

Comment (wochinge) [reply rasa/core/schemas/stories.yml:50]: depends a bit on what was done here https://github.com/RasaHQ/rasa/issues/6430

Comment (tmbo) [reply rasa/core/schemas/stories.yml:50]: user messages for story tests (previously e2e tests) are here https://github.com/RasaHQ/rasa/blob/master/rasa/core/schemas/stories.yml#L29

Comment (wochinge) [reply rasa/core/schemas/stories.yml:50]: Yep, found them, thanks! There will be merge conflicts once `master` is merged into `e2e` either way, so I think it's fine to resolve this then.

**Comment by wochinge** [comment]
@evgeniiaraz I'll wait to for tests to pass to make sure we didn't break anything

**Pull request merged**